frontend/lib/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -612,7 +613,8 @@ const RawElementNodeRenderer = (
     }
 
     case "skeleton": {
-      return <AppSkeleton />
+      const skeletonProto = node.element.skeleton as SkeletonProto
+      return <AppSkeleton element={skeletonProto} />
```

Comment:
I think we should use `<Skeleton />` here instead of `<AppSkeleton />`. See the comment here: https://github.com/streamlit/streamlit/pull/8074/files#r1486792887

Reply:
I think we might need to add another internal flag to skeleton proto so that it can be used as app skeleton or element skeleton. We still need the usage of the app skeleton for the initial loading screen.

Reply:
Added a "style" enum to the skeleton proto so it can switch between App and Plain styles.